repository/rest-proxy/operator/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: rest-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rest-proxy
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: rest-proxy
+    spec:
+      containers:
+      - name: rest-proxy
+        image: mesosphere/cp-kafka-rest:5.3.0
+        command: ["/etc/confluent/docker/run"]
+        resources:
+          requests:
+            memory: {{ .Params.MEMORY }}
+            cpu: {{ .Params.CPUS }}
+          limits:
+            memory: {{ .Params.MEM_LIMIT }}
+            cpu: {{ .Params.CPUS_LIMIT }}
+        ports:
+        - containerPort: {{ .Params.REST_PROXY_PORT }}
+          name: rest-proxy
+        env:
+        - name: KAFKA_REST_SCHEMA_REGISTRY_URL 
+          value: {{ .Params.SCHEMA_REGISTRY_URL }}
+        - name: KAFKA_REST_HOST_NAME
+          value: {{ .Params.HOST_NAME }}
+        - name: KAFKA_REST_BOOTSTRAP_SERVERS
+          value: {{ .Params.KAFKA_BOOTSTRAP_SERVERS }}
+        livenessProbe:
+          initialDelaySeconds: 30
```

Comment:
let's parametrize these options, with the default being these ones. As bootstrap can take time depending on the state of Kafka

Reply:
Done. Added parameters to configure this.